yarn.lock

```diff
@@ -1054,6 +1054,11 @@ ansi-regex@^4.1.0:
   resolved "https://registry.yarnpkg.com/ansi-regex/-/ansi-regex-4.1.0.tgz#8b9f8f08cf1acb843756a839ca8c7e3168c51997"
   integrity sha512-1apePfXM1UOSqw0o9IiFAovVz9M5S1Dg+4TrDwfMewQ6p/rmMueb7tWZjQ1rx4Loy1ArBggoqGpfqqdI4rondg==
 
+ansi-regex@^5.0.0:
```

Comment:
Since you didn't install any new packages, you shouldn't have any changes in your `yarn.lock` file. A quick and easy fix is just to delete this file and call `yarn install` again.